sct.py

```diff
@@ -612,23 +615,39 @@ def send_email(test_id=None, test_status=None, start_time=None, email_recipients
         start_time = format_timestamp(int(start_time))
     testrun_dir = get_testrun_dir(test_id=test_id, base_dir=logdir)
     if testrun_dir:
+        with open(os.path.join(testrun_dir, 'test_id'), 'r') as file:
+            test_id = file.read().replace('\n', '')
```

Comment:
Pretty sure that `.strip()` fits better here.

Reply:
agree